src/audio/eq_iir/CMakeLists.txt

```diff
@@ -5,4 +5,4 @@ if(CONFIG_IPC_MAJOR_3)
 	add_local_sources(sof eq_iir_ipc3.c)
 elseif(CONFIG_IPC_MAJOR_4)
 	add_local_sources(sof eq_iir_ipc4.c)
-endif()
+endif()
```

Comment:
Is your editor misconfigured?
- It removed the EOL
- It didn't show that to you in git status